### PR TITLE
CMO 2019 Week 1

### PR DESCRIPTION
**Pull request by joshukraine**

### CODE REVIEW

File: source/blog/articles/2019-06-fast-and-furious.html.markdown [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
...at happened today was that Ben and I accidentally "stole" this girl's phone. This is how it happened…. While we were giving these two girls a survey, apparently during the exchange of us giving them Good and Evils and one of the girls handing back the clipboard, she accidentally handed over her phone as well. I thought it was Ben's phone and he thought it was mine. So a couple people later, after we had finished all our surveys, he handed me the phone and said, "Here's your phone." I said, "That's not my phone; I thought it was yours." Then we both looked at each other and were like "Oh NOOO!" So we started walking back trying to find the girls, and praise the Lord, the girl called her phone! Ben answered it, we were able to find her and give back the phone! It got a bit scary there for a minute though! Anyways, we had a good time laughing about it afterwards.  Next, we were going to do the street preaching and chalk drawing, but just exactly when we got everything set up it started to rain. And it kept raining. After sitting for about 30 mins in the rain with a tarp covering all the equipment it was decided that we would have to do the preaching Saturday. It was still a great day!
```
Comment (ralphukraine) [inline source/blog/articles/2019-06-fast-and-furious.html.markdown]: Kade said "today" in the fifth sentence of this paragraph. We could change it to "that day".

Comment (ralphukraine) [reply source/blog/articles/2019-06-fast-and-furious.html.markdown]: Also we might want to delete a period after "This is how it happened".

File: source/blog/articles/2019-06-fast-and-furious.html.markdown [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
... the second village we showed the film at was that when we arrived at the place where we were to show the film at, it was still locked. We waited for a while, but no one came to unlock it we were trying to call our contacts, but no one was answering. Right about that time, two ladies and their kids came walking up to see the film. When they saw that we needed someone to come unlock the door, they started calling around to see if they could find somebody with a key. Sure enough they did, and within a half-hour the door was unlocked!  What was cool though is that every time we enter a village we are always the outsiders so to speak. But while they were helping us it was like they were on our side we were one of them! Also, I had so much fun playing with their little kids! Camping went well despite the fact that it rained most of the night. Thankfully, I don't think any of us got that wet. Today we went and handed film invitations for next Sunday at a village a little further down the road. We got that knocked out and then made it safely back to L'viv. We are all praying that God will heal Josh quickly so that he can be back with us. God is faithful. Onward we march to make Him known!
```
Comment (ralphukraine) [inline source/blog/articles/2019-06-fast-and-furious.html.markdown]: In the second sentence, we might want to add a period to split the sentence after "no one came to unlock it". We might also want to change "today" towards the end of the paragraph to "Monday".

Comment (joshukraine) [reply source/blog/articles/2019-06-fast-and-furious.html.markdown]: Great! Done.

**Pull request merged**